pages/linux/hardinfo.md

```diff
@@ -0,0 +1,7 @@
+# hardinfo
+
+> Show hardware information in GUI window.
+
+- Start hardinfo:
```

Comment:
Perhaps we could include another example or two? `hardinfo --help` outputs this for me: https://starbeamrainbowlabs.com/bin/ms05

Reply:
Yes, You're right.
Actually most of the time we use `hardinfo` as GUI tool, but it also can output hardware information to stdout.
I should add the other situation even if we rarely use it.

Reply:
Ah, ok. I guess the generation of a report might be useful in an automation scenario - e.g. where one has 100 machines and needs information on all their hardware for an audit or something.

Reply:
Yes, it's a good application situation.